docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -124,6 +133,57 @@ system health. Default value of `100ms` allows 10 enrollments per second.
 Burst of enrollments to accept before falling back to the rate defined by
 `interval`.
 
+`enroll_limit.max_body_byte_size`:::
+Maximum size in bytes of the enroll API request body.
+
+`status_limit.max`:::
+Maximum number of agents that can call the status API concurrently. This setting allows the user to avoid overloading the Fleet Server from status API calls.
+
+`status_limit.interval`:::
+How frequently agents can submit status requests to the Fleet Server.
+
+`status_limit.burst`:::
+Burst of status requests to accomodate before falling back to the rate defined by interval.
+
+`status_limit.max_body_byte_size`:::
+Maximum size in bytes of the status API request body.
+
+`upload_start_limit.max`:::
+Maximum number of agents that can call the uploadStart API concurrently. This setting allows the user to avoid overloading the Fleet Server from status API calls.
+
+`upload_start_limit.interval`:::
+How frequently agents can submit file start upload requests to the Fleet Server.
+
+`upload_start_limit.burst`:::
+Burst of file start upload requests to accomodate before falling back to the rate defined by interval.
+
+`upload_start_limit.max_body_byte_size`:::
+Maximum size in bytes of the uploadStart API request body.
+
+`upload_end_limit.max`:::
+Maximum number of agents that can call the uploadEnd API concurrently. This setting allows the user to avoid overloading the Fleet Server from status API calls.
```

Comment:
```suggestion
Maximum number of agents that can call the uploadEnd API concurrently. This setting allows the user to avoid overloading the Fleet Server from uploadEnd API calls.
```